packages/sync/src/class-defaults.php

```diff
@@ -1176,4 +1182,67 @@ public static function is_multi_network() {
 	 * @var int 1 for true.
 	 */
 	public static $default_full_sync_sender_enabled = 1; // Should send full sync items.
+
+	/**
+	 * Default Full Sync config
+	 *
+	 * @var array list of module names.
+	 */
+	public static $default_full_sync_config = array(
+		'constants'          => 1,
+		'functions'          => 1,
+		'options'            => 1,
+		'updates'            => 1,
+		'themes'             => 1,
+		'users'              => 1,
+		'terms'              => 1,
+		'posts'              => 1,
+		'comments'           => 1,
+		'term_relationships' => 1,
+	);
+
+	/**
+	 * Send all Items
+	 *
+	 * @var Array chunk limmits per request request.
+	 */
+	const FULL_SYNC_ALL = array(
+		'chunk_size' => 100,
+		'max_chunks' => 1,
+	);
+
+	/**
+	 * Default Full Sync max objects to send on a single request.
+	 *
+	 * @var array list of module => max.
+	 */
+	public static $default_full_sync_limits = array(
+		'constants'          => self::FULL_SYNC_ALL,
+		'functions'          => self::FULL_SYNC_ALL,
+		'options'            => self::FULL_SYNC_ALL,
+		'network_options'    => self::FULL_SYNC_ALL,
+		'updates'            => self::FULL_SYNC_ALL,
+		'themes'             => self::FULL_SYNC_ALL,
+		'users'              => array(
+			'chunk_size' => 100,
+			'max_chunks' => 10,
+		),
+		'terms'              => array(
+			'chunk_size' => 1000,
+			'max_chunks' => 10,
+		),
+		'posts'              => array(
+			'chunk_size' => 100,
```

Comment:
This is the same as `self::FULL_SYNC_ALL`

Reply:
good eye, but conceptually different since here we don't want to send them all in one request.